scripts/rook.sh

```diff
@@ -123,7 +132,9 @@ function teardown_rook() {
 	kubectl delete -f "${ROOK_URL}/toolbox.yaml"
 	kubectl delete -f "${ROOK_URL}/cluster-test.yaml"
 	kubectl delete -f "${ROOK_URL}/operator.yaml"
+	kubectl delete -f "${ROOK_URL}/crds.yaml" --ignore-not-found
```

Comment:
I am really not sure what happens here... does `kubectl` succeed if a non-existent file it used?

Reply:
The CRDs are installed if the version of Rook is >1.5, while deleting we are deleting it without checking the version, so in case if its on lesser version of Rook, we are saying to ignore the error. Is that clear @nixpanic

Reply:
what I wonder about:

- does `--ignore-not-found` _only_ relate to the objects that are non-existent in the .yaml file
- does `--ignore-not-found` _also_ apply when passing a file that does not exist

Reply:
it should be applied to the objects in the yaml file and if path does not exit, it will be general error like file not found , but I was in an impression that we dont have to worry about that error.